index.bs

```diff
@@ -1152,16 +1163,21 @@ dictionary FilePickerAcceptType {
 dictionary FilePickerOptions {
     sequence<FilePickerAcceptType> types;
     boolean excludeAcceptAllOption = false;
+    USVString id;
+    StartInDirectory? startIn;
 };
 
 dictionary OpenFilePickerOptions : FilePickerOptions {
     boolean multiple = false;
 };
 
 dictionary SaveFilePickerOptions : FilePickerOptions {
+    USVString? suggestedName;
```

Comment:
Why nullable? The prose below only has "if ... is specified", doesn't handle the specified-but-null case.